### DISCUSSION TITLE
Supports Relative mouse

### DISCUSSION THREAD

**Discussion by pppaapp** [rustdesk/rustdesk#4058] [Q&A] [answered]
### Description

Some apps (e.g., AutoCAD) require relative mouse input. 

Omnissa Horizon supports this mode and it’s highly useful.

### Impact

Relative mouse input is essential for certain apps and games (Unreal Engine based). 

Users will benefit significantly.

**Comment by pppaapp** [comment]
Looks like I wasn't the only one who wanted this feature.
[#2615](https://github.com/rustdesk/rustdesk/discussions/2615)


Are there any plans to support this?

**Comment by juangea** [comment]
Yep. some apps like Embergen have its viewport go crazy, and I think its because of this

**Comment by lionants02** [comment]
Relative Mouse Mode in Chrome Remote Desktop doesn't respond very well.
Self-hosting When using h 264, responsiveness is very good. If available, this feature could pave the way for remote gaming groups.

**Comment by Torbjorn-Zhang** [comment]
2024 Now, and still does not find out the final solution. Relative mouse is absolutely the final obstacle to stop me from total RDP gaming (such as Minecraft) and it has been always a problem for a decade. PPPLLLZZZ give us the relatively control :(

**Comment by wtblife** [comment]
Enabling this in chrome remote desktop seems to be the only way I'm able to use some applications for game development remotely when I'd much rather be using rustdesk. I would love this feature.

**Comment by SleepyPrince** [comment]
+1 for relative mouse support

**Comment by AndyJames11** [comment]
+1

**Comment by jss69er** [comment]
+1 for relative mouse support

**Comment by YuChuXi** [comment]
+1

**Comment by deShiko1974** [comment]
+1

**Comment by ImmortalMan01** [comment]
+1

**Comment by YuChuXi** [comment]
+1

**Comment by JpegN** [comment]
+1

**Comment by busy10** [comment]
+1

**Comment by rustdesk** [comment, marked as answer]
https://x.com/rustdesk/status/2009485520146235553
